src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -529,6 +529,24 @@ void get_num_streams(const int streams, const std::shared_ptr<ov::Model>& model,
     executor_config._threadsPerStream = executor_config._streams_info_table[0][THREADS_PER_STREAM];
 }
 
+void set_config_streams_0(Config& config) {
```

Comment:
Lets move this function on threading abstraction level as long as there is nothing plugin specific.
As plugin user I only want to force StreamExecutor to reuse current thread for execution w/o any details how the tables are implemented.